### PR TITLE
Add new query types, disclaimer information, and minor performance-related changes

### LINKED ISSUE #9: Get list of FASTag issuers
Get VPA suffixes of all FASTag issuers.

- [NPCI Issuer list](https://www.npci.org.in/what-we-do/netc-fastag/live-members) - [UPI Handles](https://www.npci.org.in/PDF/npci/netc/Bank-UPI-Handels.pdf)
- [FASTag recharges using BHIM](https://www.bankbazaar.com/driving-licence/fastag-recharge-through-bhim-upi.html)

### PR DESCRIPTION
**Pull request by srikanthlogic**

### CODE REVIEW

**Comment by srikanthlogic** [comment]
Fixes #9 Looks like the other issuers haven't enabled UPI recharge yet

**Comment by qurbat** [comment]
This should be good to go now. Let me know once you have had a chance to review and I'll merge it. Thanks for the help @srikanthlogic =)

File: data/gpay_suffixes.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+axisbank
```
Comment (srikanthlogic) [inline data/gpay_suffixes.txt]: This is fastag_suffixes.txt right? no gpay_suffixes

File: upi-recon.py [hunk 2]
```diff
@@ -22,29 +22,34 @@
...
+                     upi-recon.py -v <vpa> (query a single UPI VPA)
+                     upi-recon.py -w <word> (query a single word)
+                     upi-recon.py -f <vehicle_number> (query a vehicle number to check name of owner of the linked FASTag)
 """
 
 with open("data/general_vpa_suffixes.txt", "r") as suffix_file:
-    upi_suffix_dict = suffix_file.read().splitlines() # read all suffixes into a list
-suffix_file.close()
+    upi_suffix_dict = suffix_file.read().splitlines() #  read all suffixes into a list
+    suffix_file.close()
```
Comment (srikanthlogic) [inline upi-recon.py]: `with` autocloses file no?

File: data/gpay_suffixes.txt (hunk 1 above)
Comment (srikanthlogic) [reply data/gpay_suffixes.txt]: ``okaxis``
``okhdfcbank``
``okicici``
``oksbi``

**Comment by qurbat** [comment]
@srikanthlogic looks good, except, why not close file once list has been initialised?

File: upi-recon.py (hunk 2 above)
Comment (qurbat) [reply upi-recon.py]: Correct. You're right. Good to go.

**Pull request merged**